msal/authority.py

```diff
@@ -58,11 +53,6 @@ class Authority(object):
     """
     _domains_without_user_realm_discovery = set([])
 
-    @property
-    def http_client(self):  # Obsolete. We will remove this in next major release.
```

Comment:
This looks like a breaking change.

Reply:
Technically yes, but there should be no need for an app to make a dependency (i.e. monkeypatching) MSAL's `authority.http_cache`. Let's use this opportunity to make this intention clear. Internally, This `authority` module uses its own `_authority` member.

Reply:
What is `_authority` memeber?